addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -228,3 +228,36 @@ img.o_forum_avatar_big {
 .website_forum {
     margin-bottom: $spacer;
 }
+
+.forums main {
+    background: $gray-100;
+}
+.o_forum_card {
+    overflow: hidden;
+    padding-top: 56%;
```

Comment:
I suppose you want 16:9 ratio, so write `percentage(9/16);`
But in fact, I think you can achieve the same behavior without any css using bootstrap embed classes: https://getbootstrap.com/docs/4.3/utilities/embed/#aspect-ratios
Use them on the cols and you won't need absolute positioning in your scss.